rpc/src/v1/impls/light/parity.rs

```diff
@@ -415,6 +415,19 @@ impl Parity for ParityClient {
 		Box::new(self.fetcher().header(id).and_then(from_encoded))
 	}
 
+	fn block_receipts(&self, number: Trailing<BlockNumber>) -> BoxFuture<Vec<Receipt>> {
+		// Note: Here we treat `Pending` as `Latest`.
+		//       Since light clients don't produce pending blocks
+		//       (they don't have state) we can safely fallback to `Latest`.
+		let id = match number.unwrap_or_default() {
+			BlockNumber::Num(n) => BlockId::Number(n),
+			BlockNumber::Earliest => BlockId::Earliest,
+			BlockNumber::Latest | BlockNumber::Pending => BlockId::Latest,
```

Comment:
Since there is no Pending state, I'd say either return an empty vector or keep the [deprecation log](https://github.com/paritytech/parity-ethereum/blob/6e62d77e4d6211fca1b66a34f3b5047f3a406e77/rpc/src/v1/impls/light/eth.rs#L80) from `num_to_id`.